guide/glossary/index.md

```diff
@@ -206,6 +206,10 @@ Same as xpub, however the y denotes that this xpub belongs to a wallet that is f
 
 Same as ypub though the z denotes it is an extended public key from a segregated witness enabled wallet following [BIP84](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki).
 
+### Fiat
+
+Fiat money is a type of currency that is issued and backed by a government. It has value because the government says it does, and people are willing to accept it as payment for goods and services. Examples of fiat money include the US dollar, Indian rupees, the euro, and the Japanese yen. Unlike physical commodities like gold, silver or copper, fiat money is not backed by a physical commodity and its value is not directly tied to the value of a commodity. Instead, it is based on trust in the issuing government and its ability to maintain a stable currency. To learn more, visit [Wikipedia](https://en.wikipedia.org/wiki/Fiat_money).
```

Comment:
```suggestion
Fiat money is a type of currency that is issued and backed by a government. It has value because the government says it does, and people are willing to accept it as payment for goods and services. Examples of fiat money include the US dollar, Indian rupees, the euro, and the Japanese yen. The value of fiat money is not backed by, or tied to a commodity such as gold, silver or copper. Instead, it is based on trust in the issuing government and its ability to maintain a stable currency. To learn more, visit [Wikipedia](https://en.wikipedia.org/wiki/Fiat_money).
```

Reply:
Slight simplification, avoiding repetition.